tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -40,6 +40,10 @@ TEST_F(WholeFileTest, BezierHullShader) {
   runWholeFileTest("bezier.hull.hlsl2spv");
 }
 
+TEST_F(WholeFileTest, BezierDomainShader) {
+  runWholeFileTest("bezier.domain.hlsl2spv");
+}
+
```

Comment:
It would be nice to add dedicated tests for each feature newly supported, like:
* semantic.position.ds.hlsl
* semantic.domain-location.ds.hlsl

We've added hull and domain shaders now but didn't go over the previously supported SV semantics to see how they should be translated in hull and domain shaders. But that's a separate task though.

Reply:
Will add. Thanks